src/coreComponents/physicsSolvers/PhysicsSolverBase.cpp

```diff
@@ -127,11 +127,11 @@ PhysicsSolverBase::PhysicsSolverBase( string const & name,
 void PhysicsSolverBase::postInputInitialization()
 {
   m_solverStatistics.setOutputFilesName( getName() );
-  m_solverStatistics.makeDir( m_writeStatisticsCSV >= 2 );
+  m_solverStatistics.makeDir( m_writeStatisticsCSV >= 1 );
 
   getIterationStats().setTableName( getName() );
-  getIterationStats().setLogOutputState( m_writeStatisticsCSV >= 1 );
-  getIterationStats().setCSVOutputState( m_writeStatisticsCSV >= 2 );
+  getIterationStats().setLogOutputState( true );
```

Comment:
No log level for that?

Just suggesting, feel free to ignore / adapt
```suggestion
  getIterationStats().setLogOutputState( isLogLevelActive< Statistics >( this->getLogLevel() ) );
```

Reply:
By default it always printed iteration stats at the end of the simulation and I think we better preserve that. I recently made it to print iteration stats after every cycle for log level >= 2 which is useful for convergence investigations. That logInfo:: Statistics has a different flavor, I was also confused yesterday, maybe we should rename it to make more specific.